camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java

```diff
@@ -356,7 +362,9 @@ public int compare(FileStatus f1, FileStatus f2) {
     startTiming("commit");
 
     // Send Tracking counts to Kafka
-    sendTrackingCounts(job, fs, newExecutionOutput);
+    String etlCountsClassName = props.getProperty(ETL_COUNTS_CLASS, ETL_COUNTS_CLASS_DEFAULT);
+    Class<? extends EtlCounts> etlCountsClass = (Class<? extends EtlCounts>) Class.forName(etlCountsClassName);
```

Comment:
The purpose of this is to be able to use class EtlCountsForUnitTest, instead of EtlCounts, in unit test.